dev/reference/configuration/tikv-server/configuration-file.md

```diff
@@ -0,0 +1,1050 @@
++ Default value: 10s
++ Minimum value: `0`
+
+### `snap-mgr-gc-tick-interval`
+
++ The time interval at which the recycle of expired snapshot files is triggered. `0` means that this feature is not enabled.
++ Default value: 5s
++ Minimum value: `0`
+
+### `snap-gc-timeout`
+
++ The longest time for which a snapshot file is saved
++ Default value: 4h
++ Minimum value: `0`
+
+### `lock-cf-compact-interval`
+
++ The time interval at which TiKV triggers a manual compaction for the Lock Column Family
++ Default value: 256MB
++ Default value: 10m
++ Minimum value: `0`
+
+### `lock-cf-compact-bytes-threshold`
+
++ The size out of which TiKV triggers a manual compaction for the Lock Column Family
++ Default value: 256MB
++ Minimum value: `0`
++ Unit: MB
+
+### `notify-capacity`
+
++ The longest length of the Region message queue.
++ Default value: `40960`
++ Minimum value: `0`
+
+### `messages-per-tick`
+
++ The maximum number of messages processed per round
++ Default value: `4096`
++ Minimum value: `0`
+
+### `max-peer-down-duration`
+
++ The longest inactive time allowed for a peer. A peer with timeout is marked as `down`, and PD tries to delete it later.
++ Default value: 5m
++ Minimum value: `0`
+
+### `max-leader-missing-duration`
+
++ The longest time allowed for a peer to be in the leaderless state. If this value is exceeded, the peer verifies with PD whether the peer has been deleted.
++ Default value: 2h
++ Minimum value: greater than `abnormal-leader-missing-duration`
+
+### `abnormal-leader-missing-duration`
+
++ The longest time allowed for a peer to be in the leaderless state. If this value is exceeded, the peer is seen as abnormal and marked in metrics and logs.
++ Default value: `10m`
++ Minimum value: greater than `peer-stale-state-check-interval`
+
+### `peer-stale-state-check-interval`
+
++ The time interval at which the check for whether a peer is in the leaderless state is triggered
++ Default value: 5m
++ Minimum value: greater than `2 * election-timeout`
+
+### `leader-transfer-max-log-lag`
+
++ The maximum number of missing logs allowed by the transferee in an attempt to transfer Raft leadership
++ Default value: `10`
++ Minimum value: `10`
+
+### `snap-apply-batch-size`
+
++ The memory cache size when the imported snapshot file needs to be written into disk
++ Default value: 10MB
++ Minimum value: `0`
++ Unit: MB
+
+### `consistency-check-interval`
+
++ The time interval at which the consistency check is triggered. `0` means that this feature is not enabled.
++ Default value: 0s
++ Minimum value: `0`
+
+### `raft-store-max-leader-lease`
+
++ The longest trusted period of a Raft leader
++ Default value: 9s
++ Minimum value: `0`
+
+### `right-derive-when-split`
+
++ The direction in which a Region is split, left or right. When this value is set to `true`, the sequence numbers of the split Regions are `1` and `0`. Otherwise, the sequence is `0` and `1`.
```

Comment:
This is confusing to me. 
- What are "sequence numbers"?
- "Otherwise" is not clear.
- The meaning of the last two sentences is unclear.

Please check and update.

Reply:
Addressed.